src/Mappers/TransactionMapper.php

```diff
@@ -2,10 +2,15 @@
 
```

Comment:
Can we use this 'general' `TransactionMapper` also to get the new `BankTransactions` or do we need to create a new mapper if we would like to get them from Twinfield?

Reply:
I can give it a try, if you provide me with an XML file I will create / adapt the mapper.

Reply:
I used the Request and Response examples from the documentation, in this case at the bottom of [the Bank transactions documentation](https://c3.twinfield.com/webservices/documentation/#/ApiReference/Transactions/BankTransactions), under the 'XML Example' header.